config/manager/manager.yaml

```diff
@@ -47,7 +47,7 @@ spec:
         volumeMounts:
         - mountPath: /tmp/koku-metrics-operator-reports
           name: koku-metrics-operator-reports
-      serviceAccountName: koku-metrics-manager-role
+      serviceAccountName: controller-manager
```

Comment:
Question: Why make this change?

I _think_ initially, these `serviceAccountNames` were all `default` but I think we read somewhere that we shouldn't use the default, so we changed them to `koku-metrics-manager-role`.

I'm fine with changing it to `controller-manager`, but would rather not if we didn't need to.